Bottom: 81ed8a2785e568c0e18190ede2bcbf29507384fb
Top:    3bd8028a026e4eea5eed1f2ac52682172df6c6b2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 12:51:33 +0000

Refresh of push-special-case-thindraw-vs-

---

diff --git a/src/hid_draw.h b/src/hid_draw.h
index 70b59de..ef34be9 100644
--- a/src/hid_draw.h
+++ b/src/hid_draw.h
@@ -54,7 +54,6 @@ struct hid_draw_st
   void (*draw_pcb_line) (hidGC gc, LineType *line);
   void (*draw_pcb_arc) (hidGC gc, ArcType *arc);
   void (*draw_pcb_text) (hidGC gc, TextType *, Coord);
-
   void (*draw_pcb_polygon) (hidGC gc, PolygonType *poly, const BoxType *clip_box);
   void (*fill_pcb_pad) (hidGC gc, PadType *pad, bool clip, bool mask);
   void (*thindraw_pcb_pad) (hidGC gc, PadType *pad, bool clip, bool mask);
